spec/defines/director_console_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'bacula::director::console' do
+  on_supported_os.each do |os, facts|
+    context "on #{os}" do
+      let(:facts) { facts }
+
+      let(:pre_condition) { 'include bacula' }
+
+      context 'with default parameters' do
+        let(:title) { 'Monitoring' }
+
+        let(:params) do
+          {
+            password: 'monitoring_password',
+          }
+        end
+
+        it { is_expected.to contain_concat__fragment('bacula-director-console-Monitoring').with(content: <<~FRAGMENT) }
+          Console {
+              Name        = Monitoring
+              Password    = "monitoring_password"
+              CatalogACL  = *all*
+              CommandACL  = list
+          }
+        FRAGMENT
+      end
+
+      context 'with all parameters set' do
+        let(:title) { 'restricted-user' }
+
+        let(:params) do
+          {
+            password: 'A different UntrustedUser',
+            jobacl: 'Restricted Client Save',
+            clientacl: 'restricted-client',
+            storageacl: 'second-storage',
+            scheduleacl: 'weekly-schedule',
+            poolacl: 'backup-pool',
+            filesetacl: "Restricted Client's FileSet",
+            catalogacl: 'RestrictedCatalog',
+            commandacl: %w[run restore],
+            whereacl: '/',
+          }
+        end
+
+        it { is_expected.to contain_concat__fragment('bacula-director-console-restricted-user').with(content: <<~FRAGMENT) }
+          Console {
+              Name        = restricted-user
+              Password    = "A different UntrustedUser"
+              JobACL      = Restricted Client Save
+              ClientACL   = restricted-client
+              StorageACL  = second-storage
+              ScheduleACL = weekly-schedule
+              PoolACL     = backup-pool
+              FileSetACL  = Restricted Client's FileSet
+              CatalogACL  = RestrictedCatalog
+              CommandACL  = run, restore
+              WhereACL = /
+          }
```

Comment:
I took this example from:
https://www.bacula.org/5.1.x-manuals/en/main/main/Console_Configuration.html

As you can see, there is some quoting in the documentation that is not present with the module. I can't tell if it is an issue or not, maybe something to check?

Maybe we will need some serialization function, e.g. if a String contain space, we want to double-quote it, and if it has double quotes, some escaping may be required.  This can be done in a future PR, if you have info about what should / may / must be quoted, fill free to fill-in an Issue where you provide pointers.  And if you want to contribute in this area, I will be happy to help!

Reply:
I don't have any info about quoting as all I really needed was just a few lines in config to enable monitoring.
I'm sorry but I'm swamped with other work right now so don't expect more PRs from me any time soon.

Reply:
Fine, I guess that if it works for your use-case, it is a good start!